mapping.yml

```diff
@@ -0,0 +1,5 @@
+mappings:
```

Comment:
did you commit this file intentionally?

Reply:
@matthiasr you're too fast man! Adding a new test like @SuperQ recommended, committed by accident 😇